tests/eko/scale_variations/test_expanded.py

```diff
@@ -64,20 +63,50 @@ def scheme_diff(g, k, pto, is_singlet):
         the accuracy of singlet quantities is slightly worse.
         """
         if pto[0] >= 2:
-            diff = g[0] * k * a0
+            diff = g[0] * k * a0 - 2 * a1 * k * g[0]
```

Comment:
This is not higher order (~a with PTO=2) and not what we found in the notebook: $L \left(\text{a0}^2 \beta [0]-(\text{a0}-a[t])^2 \gamma [0]\right) (\text{b1} \gamma [0]-\gamma [1])$

Reply:
Indeed this is not HO because the test is not doing the full A vs B, so it's not what we have derived in that matematica NB.